ddl/constant.go

```diff
@@ -66,6 +66,7 @@ const (
 	// BackgroundSubtaskHistoryTableSQL is the CREATE TABLE SQL of `tidb_background_subtask_history`.
 	BackgroundSubtaskHistoryTableSQL = `create table tidb_background_subtask_history (
 	 	id bigint not null auto_increment primary key,
+		step int,
```

Comment:
what is this step used for?

Reply:
Stay consistent with `BackgroundSubtaskTableSQL`, see L49-65